app/spaces/[spaceid]/trustful/utils/abi.ts

```diff
@@ -0,0 +1,421 @@
+export const RESOLVER_CONTRACT_SCROLL_ABI = [
```

Comment:
This ABI can be cut down to include only the method(s) that is being used from it

Reply:
Yes. It's already working like that in each function call.

The main idea of ​​this file is to show the RESOLVER ABI easily, in case the contract is deployed again and updates the ABI. Also for pattern reasons, there are files like this in the repository, but for trustful, the function call is only getting the object that contains the function from that file.

But, based on what I said about the patterns, and the ABI reason, you think is the best option to this file. Remove the abi.ts ?